libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -979,16 +979,22 @@ AP_GPS_UBLOX::_parse_gps(void)
                         gnssCount++;
                     }
                 }
-
                 for(int i = 0; i < _buffer.gnss.numConfigBlocks; i++) {
-                    // Reserve an equal portion of channels for all enabled systems
+                    // Reserve an equal portion of channels for all enabled systems that supports it
                     if(gps._gnss_mode[state.instance] & (1 << _buffer.gnss.configBlock[i].gnssId)) {
-                        if(GNSS_SBAS !=_buffer.gnss.configBlock[i].gnssId) {
+                        if(GNSS_SBAS !=_buffer.gnss.configBlock[i].gnssId && (_hardware_generation > UBLOX_M8 || GNSS_GALILEO !=_buffer.gnss.configBlock[i].gnssId)) {
                             _buffer.gnss.configBlock[i].resTrkCh = (_buffer.gnss.numTrkChHw - 3) / (gnssCount * 2);
                             _buffer.gnss.configBlock[i].maxTrkCh = _buffer.gnss.numTrkChHw;
-                        } else {
-                            _buffer.gnss.configBlock[i].resTrkCh = 1;
-                            _buffer.gnss.configBlock[i].maxTrkCh = 3;
+                        }
+                        else {
+                            if(GNSS_SBAS ==_buffer.gnss.configBlock[i].gnssId) {
```

Comment:
The ArduPilot has a coding style.
For example, an if statement is if () { with a space between.
I think it is better to match the coding style.

https://ardupilot.org/dev/docs/style-guide.html

Reply:
As an aside, it's a huge pet peeve of mine that this is not automatically applied.

Reply:
There are commands to automatically convert sources to coding styles and their definition files.
Once this PR was merged, I wanted to convert the format with this command and PR it again.
astyle --options=Tools/CodeStyle/astylerc <filename>

Reply:
Yes, but the entire codebase isn't automatically formatted, and if you change a line in a file and run the formatter, the diff is ruined.